mkt/developers/models.py

```diff
@@ -487,3 +487,11 @@ def __unicode__(self):
 
     def __html__(self):
         return self
+
+
+class IARCRequestID(ModelBase):
```

Comment:
nit: I'd just call this `IARCRequest` (dropping the ID/_ids everywhere) but maybe that's just me.